shared/teams/index.js

```diff
@@ -1,27 +1,163 @@
 // @flow
-import Render from './render'
-import pausableConnect from '../util/pausable-connect'
-import openURL from '../util/open-url'
+import * as React from 'react'
+import {ClickableBox, Box, Icon, ScrollView, Text} from '../common-adapters'
+import {globalColors, globalMargins, globalStyles} from '../styles'
+import {isMobile} from '../constants/platform'
 
-type DispatchProps = {
+import type {IconType} from '../common-adapters/icon.constants'
+
+type HeaderButtonProps = {
+  iconType: IconType,
+  label: string,
+  onClick: () => void,
+}
+
+const marginHorizontal = isMobile ? globalMargins.tiny : globalMargins.medium
+const headerButtonBoxStyle = {
+  ...globalStyles.flexBoxRow,
+  alignItems: 'center',
+  marginLeft: marginHorizontal,
+  marginRight: marginHorizontal,
+}
+
+const HeaderButton = (props: HeaderButtonProps) => (
+  <ClickableBox onClick={props.onClick} style={headerButtonBoxStyle}>
+    <Icon type={props.iconType} style={{color: globalColors.blue}} />
+    <Text type={isMobile ? 'BodySemiboldLink' : 'HeaderLink'} style={{margin: globalMargins.tiny}}>
+      {props.label}
+    </Text>
+  </ClickableBox>
+)
+
+type HeaderProps = {
+  onCreateTeam: () => void,
+  onJoinTeam: () => void,
+}
+
+const Header = (props: HeaderProps) => (
+  <Box style={{...globalStyles.flexBoxRow, alignItems: 'center', height: 48}}>
+    <HeaderButton iconType="iconfont-new" label="Create a team" onClick={props.onCreateTeam} />
+    <HeaderButton iconType="iconfont-team-join" label="Join a team" onClick={props.onJoinTeam} />
+  </Box>
+)
+
+const Banner = ({onReadDoc, onHideBanner}) => (
```

Comment:
maybe rename `onReadDoc` to `onReadAnnouncement`, since when the blog post is available we want the banner to link to that but not the beta note

Reply:
spoke to coyne and he wants to point them to the same place (blog) since it has more info. and wants the bottom renamed to be Read more vs Read the doc